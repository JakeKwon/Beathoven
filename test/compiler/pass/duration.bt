duration p = 2/8; // 1/4
duration q;
q = 1/2;
//print("duration p is ",str_of_duration(p), "\n");
//print("duration q is ",str_of_duration(q));