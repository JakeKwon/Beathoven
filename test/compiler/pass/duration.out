duration p is 1/4
duration q is 1/2